examples/serve-inference/README.md

```diff
@@ -0,0 +1,83 @@
+This example compiles Open LLAMA-3B model and deploys the model on Trainium (Trn1)  instance
+using Ray Serve and FastAPI. It uses transformers-neuronx to shard the model across devices/neuron cores
+via Tensor parallelism. 
+
+
+## Setup 
+* Clone this repo to run the example on your local laptop:
+```
+git clone https://github.com/aws-samples/aws-samples-for-ray
+
+cd aws-samples-for-ray/examples/serve-inference
+```
+
+* Replace subnet and security-group where you intend to launch the cluster in `cluster-inference-serve.yaml`
```

Comment:
Maybe explain how to create the subnet and security group here. You can just add the link to AWS docs.